source/client/module.coffee

```diff
@@ -1,26 +1,35 @@
-class Space.accountsUi extends Space.Module
+Space.accountsUi = Space.Module.define 'Space.accountsUi',
 
-  @publish this, 'Space.accountsUi'
+  configuration: {
```

Comment:
Great use of configuration here, I like this pattern of configurable events a lot